text/3-entrees/basic-token.md

```diff
@@ -24,11 +32,13 @@ decl_storage! {
 }
 ```
 
-Because users can influence the keys in our storage map, we've chosen the `blake2_128_concat` hasher as described in the recipe on [storage maps](storage-api/storage-maps.md)s.
+Because users can influence the keys in our storage map, we've chosen the `blake2_128_concat` hasher
```

Comment:
Maybe it's worth elaborating on why user-defined keys is a problem and how the concatenating hasher addresses that?

Reply:
[Choosing a hasher](https://substrate.dev/recipes/3-entrees/storage-api/storage-maps.html#choosing-a-hasher) is explained in detail in the recipe on storage maps linked on the next line.